### PR TITLE
Article milestone1 pull request

### PR DESCRIPTION
**Pull request by ahzia**
We have worked on the first milestone for "tnw-news" project;
We build 4 sections (Header, Footer, Highlights, and Latest Funding ) for three screens (Mobile, Tablet and laptop)
Kindley Review our code.
Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+      <nav>
+        <div class="top flex">
+          <div class="blackback flex">
+            <ul>
+              <li>News</li>
+              <li>Events ▾</li>
+              <li>Business ▾</li>
+              <li>AMA's</li>
+              <li>Spaces</li>
+              <li>Terms & conditions</li>
```
Comment (henatan99) [inline index.html]: - All menu options should have a-tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+            </ul>
+            <ul>
+              <li>About</li>
+              <li>advertise</li>
+              <li>Jobs</li>
+              <li>Contact</li>
```
Comment (henatan99) [inline index.html]: - All menu options should have a-tag (You can wrap a-tags by li-tags)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+            <div><i class="fa fa-bars menu"></i></div>
+          </div>
+
+          <div class="search-menu-bg flex">
+            <ul class="flex scndlinks">
+              <li>LATEST</li>
+              <li>HARD FORK</li>
+              <li>PLUGGED</li>
+              <li>README</li>
+              <li>GROWTH QUARTERS</li>
```
Comment (henatan99) [inline index.html]: - All menu options should have a-tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+            </ul>
+            <div><i class="fa fa-search search"></i></div>
```
Comment (henatan99) [inline index.html]: - Search button needs to have a-tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+            <img
+              alt="The Next Web"
+              src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+            />
+          </div>
+          <div class="search-menu-div flex">
+            <hr />
+            <div><i class="fa fa-search search"></i></div>
+            <hr />
+            <div><i class="fa fa-bars menu"></i></div>
```
Comment (henatan99) [inline index.html]: - search and hamburger buttons needs to have a-tag

File: index.html [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+          <p>
+            Calling young digital talent in the Netherlands – Be recognized in
+            the T500 →
+          </p>
+        </div>
+      </nav>
+    </header>
+
+    <section class="highlights">
+      <div class="highlight-one">
```
Comment (henatan99) [inline index.html]: Change this tag to article

File: index.html [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+        <div class="highlight-top">
+          <img
+            alt="The Next Web"
+            src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+          />
+        </div>
+        <div class="highlight-center">
+          <p>
```
Comment (henatan99) [inline index.html]: This tag should be hx-tag(h1 or h2 or..)  and should have a-tag also

File: index.html [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+          </p>
+        </div>
+        <div class="highlight-bottom">
+          <p>by Ivan Mehta 1 hour ago <i class="fa fa-share-alt"></i> 6</p>
+        </div>
+      </div>
+      <div class="highlight-two">
+        <div class="highlight-top"></div>
+        <div class="highlight-center">
+          <p>
```
Comment (henatan99) [inline index.html]: Change this p-tag to hx-tag and also should have a-tag

File: index.html [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+            Opal cuts off your apps' internet access to help you stop
+            downloading
+          </p>
+        </div>
+        <div class="highlight-bottom">
+          <p>by Ivan Mehta 1 hour ago <i class="fa fa-share-alt"></i> 6</p>
+        </div>
+      </div>
+      <div class="highlight-two">
```
Comment (henatan99) [inline index.html]: this div should be replaced by article tag

File: index.html [hunk 10]
```diff
@@ -0,0 +1,292 @@
...
+          <p>
+            Opal cuts off your apps' internet access to help you stop
+            downloading
+          </p>
+        </div>
+        <div class="highlight-bottom">
+          <p>by Ivan Mehta 1 hour ago <i class="fa fa-share-alt"></i> 6</p>
+        </div>
+      </div>
+      <div class="highlight-three">
```
Comment (henatan99) [inline index.html]: This div should be replaced by article tag

File: index.html [hunk 11]
```diff
@@ -0,0 +1,292 @@
...
+        <div class="highlight-top"></div>
+        <div class="highlight-center">
+          <p>
```
Comment (henatan99) [inline index.html]: Changes this p-tag with hx-tag and also should have a-tag

File: index.html [hunk 12]
```diff
@@ -0,0 +1,292 @@
...
+        <div class="latest-funding-title flex">
+          <div class="latest-funding-heading">
+            <h1>Latest funding rounds on index.co</h1>
+          </div>
+        </div>
+        <ul class="latest-funding-nav flex">
+          <li>See more</li>
+          <li>Get updates by email</li>
+          <li>App info</li>
+        </ul>
```
Comment (henatan99) [inline index.html]: - This ul should be wrapped by a nav element 
- All menu options (li items here) should have a-tag

File: index.html [hunk 13]
```diff
@@ -0,0 +1,292 @@
...
+              <img
+                class="story-investmentCard-company-image"
+                alt=""
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1346411998544879617%2FQQiDnIXT.png&amp;signature=6a474c1592a800091fe31b2d54a18545"
+              />
+            </div>
+            <div class="funding-card-body">
+              <div class="flex funding-card-body-top">
+                <p><i class="fa fa-tag"></i> Telecommunications</p>
+                <p><i class="fa fa-briefcase"></i> Expedition growth capital</p>
```
Comment (henatan99) [inline index.html]: These p-tags should also have a-tags. Please extrapolate this comment to the other latest-funding section div elements

File: index.html [hunk 14]
```diff
@@ -0,0 +1,292 @@
...
+              </div>
+              <hr />
+              <p class="sm-text">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+                Molestias, exercitationem.
+              </p>
```
Comment (henatan99) [inline index.html]: This p-tag should also have a-tag. Please extrapolate this comment to the other latest funding section div elements

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes Required #:recycle: 
Hey there, you have done a lot so far, yet there are some changes you need to make before the project gets approved.
### Please be aware that this is your first review and you will be left with only two re-reviews. If your project cannot be approved in the second re-review(third review), you will repeat the project. 
## To Highlight 
### General requirements 
- PR has a descriptive title and a nice summary 
- Cool commit messages 
- Correct Git flow 
### Project-specific 
- Nicely behaving responsive design for 1024px min and 768px min breakpoints
- Mobile-first design approach is followed
- Footer and header elements are properly aligned using flex 
- Grid is nicely employed in the highlights section 

## To Improve 
- In the Highlight section, the highlight center p-tag texts positioning is not as the original design in the 768px min and 1024px min views
- For good semantic's sake, you should make the 'highlight-one', 'highlight-two' and 'highlight-three' elements article elements instead of div
- You should change the p-tags to hx-tags (headings) in the highlight-one, -two, and -three elements, and also should have a-tags, please refer to the inline comments 
- The logo 'TNW' does not have a-tag, so it should have 
- Header should stick to the top when the user scrolls the page 
- Use a-tag for all menu items, search button, and hamburger button (Please refer inline comments) 
- In mobile view, the yellow section(top-cta class) should have big top and bottom padding when it is mobile view 
- The hamburger button should disappear when it is tablet view 
- The date should appear over the highlight-one element, at the top right corner
- Please see inline comments for the latest-funding section comments 
- You forget the vertical-line(pipe) character in the latest funding nav element for tablet and desktop views 
- The social media icons in the footer should have a-tag

### Please make the requested changes and submit for re-review 

### Once again, Please be aware that this is your first review and you will be left with only two re-reviews. If your project cannot be approved in the second re-review(third review), you will repeat the project. 

###Happy Coding :computer:

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Status: Change Requested

Amazing job guys on implementing almost all changes the previous TSE 
asked to make however you still have small changes to make and you are ready for another milestone

- Hamburger button should disappear on the tablet
- The first article appears bigger on the left, and its title is bigger on tablet
- The other 2 articles appear smaller on the right, the second on top of the third on the tablet

### Implement those small changes and submit another CRR and remember after this review you still have one CR only so work accordingly


# Happy coding image

Best,

Lydia Ingabire

**Comment by blakbox23** [comment]
@DeliceLydia  The hamburger dissapears on tablet view. This is a picture of the view on my end. Is there something I'm missing? This is the tablet view

Menu

**Comment by DeliceLydia** [comment]
@blakbox23 thanks for reaching out it might be my computer with a bigger 
screen don't mind about that then move on

tnw

**Comment by blakbox23** [comment]
I have the changed the sizes of the highlights as well as the font sizes for the two articles 
@DeliceLydia

**Comment by DeliceLydia** [comment]
Okey submit for another CR and hopefully you are good to go

File: style.css [hunk 15]
```diff
@@ -0,0 +1,565 @@
...
+
+  .highlights {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    gap: 10px;
+    height: 85vh;
+    margin-top: 90px;
+    width: 96vw;
+  }
```
Comment (henatan99) [inline style.css:405]: Please correct this grid parameter as it is giving a faulty grid as shown in the screenshot. 
Screenshot from 2021-04-01 00-17-39

File: index.html [hunk 16]
```diff
@@ -0,0 +1,365 @@
...
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <nav class="sticky">
+        <div class="top flex">
+          <div class="blackback flex">
```
Comment (henatan99) [inline index.html:20]: This tag should be a nav

File: index.html [hunk 17]
```diff
@@ -0,0 +1,365 @@
...
+            </a>
+          </div>
+          <div class="search-menu-div flex">
+            <hr />
+            <a href="#"><i class="fa fa-search search"></i></a>
+            <hr />
+            <a href="#"><i class="fa fa-bars menu"></i></a>
+          </div>
+
+          <div class="search-menu-bg flex">
```
Comment (henatan99) [inline index.html:52]: this tag should  be a nav

**Review by henatan99 — APPROVED** [review]
# Project Approved :100:

## Congratulations your project is approved :+1: :partying_face: . Thank you for implementing the requested changes from the last review. 

For optional improvements you can refer the below comments 
## To Highlight 
- Nice responsive design 
- Using a-tags wherever required and nice use of semantic tags

## [OPTIONAL] Improvements 
- The highlights section grid is faulty in tablet and desktop views. Please check the inline comment 
- Change both nav menus to nav tag from div tag. See inline comments 

### You can now merge the PR and submit the mandatory project completion form 
### Happy Coding :computer: :computer_mouse:

**Pull request merged**